include/psa/crypto_config.h

```diff
@@ -1574,7 +1574,7 @@
  *
  * \note  This feature is incompatible with insecure block cipher,
  *        MBEDTLS_DES_C, and cipher modes which always require decryption
- *        operation, MBEDTLS_CIPHER_MODE_CBC, MBEDTLS_CIPHER_MODE_XTS and
+ *        operation, MBEDTLS_CIPHER_MODE_CBC and MBEDTLS_CIPHER_MODE_XTS and.
```

Comment:
Typos: extra "and" (1st one), extra period (end of line). Can we just go back to the text as it was before this PR?